app/src/main/kotlin/com/wire/android/util/ColorUtil.kt

```diff
@@ -1,13 +1,20 @@
 package com.wire.android.util
 
+import android.annotation.SuppressLint
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.ui.graphics.Color
+import com.wire.android.ui.common.colorsScheme
+import com.wire.android.ui.theme.WireColorSchemeTypes
+import com.wire.android.ui.theme.wireColorScheme
 import com.wire.kalium.logic.data.id.ConversationId
 import kotlin.math.absoluteValue
 
 // TODO: remove once the solution is implemented
+@SuppressLint("PackagePrivateId")
 @Suppress("MagicNumber")
-private val conversationColors = listOf(0xFFFF0000, 0xFF00FF00, 0xFF0000FF)
+private val conversationColors = WireColorSchemeTypes.light.groupAvatarColors
```

Comment:
```suggestion
private val conversationColors = WireColorSchemeTypes.currentTheme.groupAvatarColors
```

With accessing the `currentTheme`, we can abstract and be independent from the current dark/light =)

Reply:
@yamilmedina eh no. Current theme is @composable and the function is not

Reply:
mmm yeah actually, that I checked out the branch, no quick/direct workarounds to that, since we need to be in "compose" world to know if we are on dark or light mode. Maybe using the light is ok for now, since we are not fully supporting dark mode

Reply:
Right, we shouldn't pass the colors in models as they should be theme-dependent.
We can pass some kind of colorId or just remove all val groupColorValue: ULong, and instead just make getConversationColor https://github.com/composable and use it directly in Composables? 🤔

Reply:
hey @tmspzz here is a proposal to workaround the composable issue, on the lines from @saleniuk  https://github.com/wireapp/wire-android-reloaded/compare/fix/AR-1531-group-avatar-colors...fix/refactor-proposal-colors?expand=1#diff-b6e3e8db5ff1d951da83c4422a23a125da5b831b5c352fac8eaa2c2a00fe65fd 😃

Reply:
Presented solution has one drawback - what if `groupAvatarColors` list size is different on light and dark theme? We would need some additional abstraction level to convert such color index to the actual color for each theme 😉